loop.cpp

```diff
@@ -37,9 +36,9 @@ void Loop::Play()
 
 void Loop::ShowPlayers()
 {
-    for (unsigned short i = 0; i < NUMBER_OF_PLAYERS; i++)
+    for (auto player : this->players)
```

Comment:
Itt is const referenciát kellene használni, hogy ne másolódjanak feleslegesen a Playerek.